src/EnergyPlus/WindowModel.hh

```diff
@@ -0,0 +1,62 @@
+#ifndef WindowModel_H
+#define WindowModel_H
+
+#include <memory>
+#include <iostream>
```

Comment:
Why do you need to include `#include <iostream>` here? it doesn't seem like you are using any IO within the header file.

Reply:
@mbadams5 I have done requested changes. Can you please review them?